README.md

```diff
@@ -6,6 +6,10 @@ Notable features:
 - No complicated cross-compiling or big library dependencies! Builds with `make` and just needs standard tools and llvm.
 - 3D graphics! nxdk includes and expands on pbkit, a library for interfacing with the Xbox GPU.
 
+Build Status
+------
+![](https://github.com/XboxDev/nxdk/workflows/Build%20Samples/badge.svg?branch=master)
```

Comment:
I think https://github.community/t5/GitHub-Actions/Badge-is-always-showing-quot-failing-quot/m-p/34807#M2019 implies that we should filter by event, to not display PRs?

Reply:
I believe filtering by branch should already be sufficient, but I added the event filter anyway.